src/CMakeLists.txt

```diff
@@ -110,17 +109,7 @@ if (PYTHONLIBS_FOUND)
     ignition-math${PROJECT_VERSION_MAJOR}
   )
 
-  # When using SWIG 3 to build a python interface there is an extra underscore between the name of
-  # the library given to swig_add_library macro and the generated .so
-  if(NOT ${SWIG_VERSION} VERSION_GREATER 4.0.0)
-    set(SWIG_PY_LIB "_${SWIG_PY_LIB}")
-    set(SWIG_PY_LIB_OUTPUT "_${SWIG_PY_LIB_OUTPUT}")
-  endif()
-
-  set_target_properties(${SWIG_PY_LIB}
-    PROPERTIES
-      OUTPUT_NAME ${SWIG_PY_LIB_OUTPUT}
-  )
```

Comment:
I thought it was related to the swig version as I commented there, but it seems that `if(NOT ${SWIG_VERSION} VERSION_GREATER 4.0.0)` should compare CMake version instead. In CI bionic is failing now.

Reply:
I've tried this at e6f285d and I locally verified the behavior for:
`bionic -- cmake 10.x`
`bionic -- cmake 21.x`

Reply:
Great! Thanks for the quick fix!